mediator/med.F90

```diff
@@ -560,16 +598,22 @@ subroutine InitializeP0(gcomp, importState, exportState, clock, rc)
        logUnit = 6
     endif
 
-    ! Obtain Verbosity
+    ! Obtain verbosity level
     call ESMF_AttributeGet(gcomp, name="Verbosity", value=cvalue, defaultValue="max", &
          convention="NUOPC", purpose="Instance", rc=rc)
     if (ChkErr(rc,__LINE__,u_FILE_u)) return
-    call ESMF_LogWrite(trim(subname)//": Mediator verbosity is "//trim(cvalue), ESMF_LOGMSG_INFO)
+    if (mastertask) then
+       write(logunit,'(a)')trim(subname)//": Mediator verbosity is set to "//trim(cvalue)
+    end if
 
-    call ESMF_AttributeGet(gcomp, name="Profiling", value=cvalue, &
-         convention="NUOPC", purpose="Instance", rc=rc)
+    ! Obtain profiling level
+    call NUOPC_CompAttributeGet(gcomp, name="Profiling", value=cvalue, isPresent=isPresent, isSet=isSet, rc=rc)
```

Comment:
We had a problem before related with setting verbosity and profiling. Does this fix it? Providing string such as low, high and max were not working and also there was some restriction in CIME side to set default value for the integer XML entiries.

Reply:
What it does fix is that I can set the profiling to the maximum number (65535) and it will work. I am printing this out to the mediator log file now.